README.md

```diff
@@ -3,3 +3,71 @@
 Core plugins and theme for https://heroeslounge.gg.
 
 To be used with https://octobercms.com/ 
+
+
+## Installation
+
+Install Virtualbox and Vagrant:
+
+On Windows, install with Chocolatey: https://chocolatey.org/
+
+    choco install virtualbox vagrant
+
+On OS X, install with Brew: https://brew.sh/
+
+    brew cask install virtualbox vagrant
+
+On Linux you should be able to figure this out yourself. ;)
+
+
+Download the database zip from the link you should have received and put it in this directory.
+
+Run Vagrant:
+
+    vagrant up && vagrant ssh
```

Comment:
my computer didn't have the bento ubuntu for vagrant, so I had to do `vagrant box add https://app.vagrantup.com/bento/boxes/ubuntu-18.04`

can you add that to the readme?

Reply:
actually I had old versions of vagrant and virtualbox that I didn't update, so maybe the newer version would have known to download it

Reply:
vagrant definitely downloads missing boxes for you.  You can test by deleting the downloaded one and seeing if it downloads it automatically now that you have upgraded vagrant

`$HOME/.vagrant.d/boxes` mac
`%USERPROFILE%\.vagrant.d\boxes` windows

Reply:
yup, you're right, it hadn't downloaded with the old version of vagrant, but when I updated it worked fine